_posts/2018-02-26-success-with-serverless.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: post
+status: publish
+published: true
+title: 'Success With Serverless'
+author:
+  display_name: Scott Davis
+  email: stdavis@utah.gov
+date: 2018-02-26 23:45:28
+categories:
+- Developer
+tags:
+- nodejs
+---
+
+In this post, I'd like to describe a recent success story that we've had using [serverless computing](https://en.wikipedia.org/wiki/Serverless_computing). This new(ish) technology has some exciting advantages over traditional solutions. We were looking for an excuse to give it a try and found a great fit.
```

Comment:
About this comment about Wikipedia--are any of the references listed in the Wikipedia article sufficient as a simple resource/link about serverless computing? I mean, it's ok to link to Wikipedia. It's not like this is a research article or something. But, again, if there's a stronger/more reliable source, it might be a good thing to link to it.